test/server/configuration-test.js

```diff
@@ -13,16 +13,19 @@ const { DEFAULT_TYPESCRIPT_COMPILER_OPTIONS } = require('../../lib/configuration
 const RunnerCtor                              = require('../../lib/runner');
 const OptionNames                             = require('../../lib/configuration/option-names');
 const consoleWrapper                          = require('./helpers/console-wrapper');
+const WARNING_MESSAGES                        = require('../../lib/notifications/warning-message');
 
 const tsConfigPath           = 'tsconfig.json';
 const customTSConfigFilePath = 'custom-config.json';
 
 const createConfigFile = (path, options) => {
     options = options || {};
-    fs.writeFileSync(path, JSON.stringify(options));
+    const content = `${pathUtil.extname(path) === '.js' ? 'module.exports = ' : ''}${JSON.stringify(options)}`;
```

Comment:
It's better to create two separate functions: `createJSONConfig`, `createJsConfig`;